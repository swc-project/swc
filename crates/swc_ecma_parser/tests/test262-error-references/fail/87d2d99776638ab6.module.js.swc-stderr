
  x Unexpected token `,`. Expected identifier or string
   ,-[$DIR/tests/test262-parser/fail/87d2d99776638ab6.module.js:1:1]
 1 | import {b,,} from 'a';
   :           ^
   `----
