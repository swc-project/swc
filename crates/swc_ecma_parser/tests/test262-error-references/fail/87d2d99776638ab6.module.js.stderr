
  x Unexpected token `,`. Expected identifier or string
   ,-[$DIR/tests/test262-parser/fail/87d2d99776638ab6.module.js:2:1]
 2 | import {b,,} from 'a';
   :           ^
   `----
